comtypes/automation.py

```diff
@@ -711,17 +711,7 @@ class tagEXCEPINFO(Structure):
         scode: int
 
     def __repr__(self):
-        return "<EXCEPINFO %s>" % (
-            (
-                self.wCode,
-                self.bstrSource,
-                self.bstrDescription,
-                self.bstrHelpFile,
-                self.dwHelpContext,
-                self.pfnDeferredFillIn,
-                self.scode,
-            ),
-        )
+        return f"<EXCEPINFO {self.wCode, self.bstrSource, self.bstrDescription, self.bstrHelpFile, self.dwHelpContext, self.pfnDeferredFillIn, self.scode}>"
```

Comment:
If the line becomes too long, I believe we could make use of a temporary variable.
```suggestion
        info = (self.wCode, self.bstrSource, self.bstrDescription, self.bstrHelpFile, self.dwHelpContext, self.pfnDeferredFillIn, self.scode)
        return f"<EXCEPINFO {info}>"
```
The contents of the tuple will actually be formatted by `ruff`, with appropriate line breaks inserted as needed.

Reply:
Good suggestion! 
Using "suggested change" is my first time.